js/packages/teams-ai/package.json

```diff
@@ -39,6 +39,8 @@
         "gpt-tokenizer": "^2.1.2",
         "json-colorizer": "^2.2.2",
         "jsonschema": "1.4.1",
+        "openai": "^4.41.0",
+        "strict-event-emitter-types": "^2.0.0",
```

Comment:
took a look at this package, I like the idea of using event emitters but could we implement it ourselves? It would be nice to avoid adding another dependency

Reply:
we could but we should probably give the original author credit somehow